python/ray/tests/aws/test_aws_batch_tag_update.py

```diff
@@ -0,0 +1,70 @@
+import threading
```

Comment:
@Gekho457 this is great! though this test isn't actually run in CI right? 

You'll have to add it into python/ray/tests/BUILD

Reply:
Ah, right – I've added a target right after  the one for`test_aws_autoscaler.py.`